src/aks-preview/HISTORY.md

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+0.5.13
```

Comment:
you do not need to add this unless you need to release it right now. the next release should pick up all the changes.

Reply:
Looking to get this into the extension sooner rather than later so its ready for when we roll the feature out to more partners.

Reply:
Can you bump the version in src/aks-preview/setup.py too then.